packages/autocomplete-types/index.d.ts

```diff
@@ -609,8 +613,20 @@ declare namespace Fig {
      * @example
      * When `requiresEqual: true` the user MUST do `--opt=value` and cannot do `--opt value`
      *
+     * @deprecated use `requiresSeparator` instead
+     *
      */
     requiresEquals?: boolean;
+    /**
+     *
+     * Signals whether one of the separators specified in parserDirectives is required to pass an argument to an option (e.g. `git commit --message[separator]"msg"`)
+     * @defaultValue false (does NOT require a separator)
+     *
+     * @example
+     * When `requiresSeparator: true` the user MUST do `--opt[separator]value` and cannot do `--opt value`
+     *
+     */
+    requiresSeparator?: boolean | string;
```

Comment:
This will just replace `requiresEquals`, if some string is passed that will be the separator to be auto-inserted.
Still needs proper docs and that feature.